### PR TITLE
Add markdown docs

### PR DESCRIPTION
**Pull request by stormpython**
Adding markdown docs.

### CODE REVIEW

**Comment by rgbkrk** [comment]
It looks like there's a merge conflict, you'll need to rebase.

**Comment by stormpython** [comment]
@rgbkrk I rebased against master and now all checks have passed. Should I add comments for the descriptions? I noticed that they were missing. I can add them quickly and push the new changes before merging.

File: packages/outputs/src/components/media/markdown.js [hunk 1]
```diff
@@ -3,7 +3,13 @@ import * as React from "react";
 import MarkdownComponent from "@nteract/markdown";
 
 type Props = {
+  /**
+   * Markdown string.
+   */
   data: string,
+  /**
+   * Mime type. Defaults to `text/markdown`.
```
Comment (rgbkrk) [inline packages/outputs/src/components/media/markdown.js:11]: This should be media type, we're switching off the mime type naming ([same as IANA](https://www.iana.org/assignments/media-types/media-types.xhtml)).

**Comment by rgbkrk** [comment]
Looks like you got those comments in for the descriptions, those are definitely helpful since they render for users.

File: packages/outputs/src/components/media/markdown.js (hunk 1 above)
Comment (stormpython) [reply packages/outputs/src/components/media/markdown.js:11]: Ok. I will make changes to the descriptions and resubmit in another PR.

**Pull request merged**